### PR TITLE
Remove deadcode in waypoint_svc_ip_address

### PR DESCRIPTION
**Pull request by howardjohn**
Minor cleanup

### CODE REVIEW

File: src/state/workload.rs [hunk 1]
```diff
@@ -242,17 +242,6 @@ impl Workload {
             service_account: self.service_account.clone(),
         }
     }
-    pub fn waypoint_svc_ip_address(&self) -> Result<Option<IpAddr>, WaypointError> {
```
Comment (zirain) [inline src/state/workload.rs:245]: is there any linter in rust like unsed function?

Comment (howardjohn) [reply src/state/workload.rs:245]: there is, i think maybe doesn't detect it due to it being public

Comment (zirain) [reply src/state/workload.rs:245]: oha, thinked it in golang-way, lower-case function means private.

**Review by keithmattix — APPROVED** [review]

**Review by ilrudie — APPROVED** [review]
lgtm

**Pull request merged**